kotsadm/web/src/components/troubleshoot/AnalyzerFileTree.jsx

```diff
@@ -101,7 +101,7 @@ class AnalyzerFileTree extends React.Component {
       dir.children ? dir.children.length : []
     });
     sortedTree.reverse(); // If something has a directory, render those first, all top level files should be at that bottom
-    this.setState({ files: sortedTree });
+    this.setState({ files: sortedTree[0].children });
```

Comment:
same here, for example the "View Files" tab, how will this affect that component

Reply:
This change can't affect "View Files" tab component because it's not shared beteween these 2 components.
Only FileTree.jsx is shared between "view files" and "support bundles".

Reply:
oh, i thought i commented on the FileTree component, my bad.